Update the Docs with Service Now Versions (#7694)
chef/automate: components/docs-chef-io/content/automate/notifications.md

    weight = 10
+++

## Notifications

> Chef Automate notifications uses webhooks to send alerts for failures of Chef Infra Client runs and InSpec compliance scans. You can send notifications to Slack channels, Slack users, or any other service that supports webhook integrations. Notifications are sent for all failures. To ignore a failure, set the `ignore_failure` attribute to `true` on the resource.

Reviewer: ```suggestion
Chef Automate notifications uses webhooks to send alerts for failures of Chef Infra Client runs and InSpec compliance scans. You can send notifications to ServiceNow, Slack channels, Slack users, or any other service that supports webhook integrations. Notifications are sent for all failures. To ignore a failure, set the `ignore_failure` attribute to `true` on the resource.
```

By default only Admins of Chef Automate may create and manage notifications.

{{< warning >}}
Notifications sent by Chef Automate do not support retries; therefore, notifications sent while Slack or the external service receiving the notification is experiencing API issues, outages, or some other unplanned downtime may never be received. Undelivered notifications are not re-sent. Attempts to send notifications do generate log messages in your Chef Automate server.
{{< /warning >}}

Chef Automate supports three types of notifications:

- ServiceNow
- Slack
- Webhooks

The steps to add different notifications to the Chef Automate is shown below:

## ServiceNow Notification

### Add a ServiceNow Notification

To add a servicenow notification for Chef Automate, follow the steps given below:

![ServiceNow notifications](/images/automate/servicenow-notifications-navigation.png)

1. In the **Settings** tab, navigate to the _Notifications_ page in the sidebar.
1. Select **Create Notification**.
1. Select **ServiceNow**.
1. Enter a unique notification name.
1. Select the failure type to be notified on from the drop-down menu. Current options are Chef Infra Client run or InSpec scan.

> 1. Get your ServiceNow webhook address by using the **What's this?** link, which opens an external servicenow site.

Reviewer: Please check if this is available or not.

Author: Checked and updated accordingly.

1. Select the **Create Notification** button to create the ServiceNow Notification.

### Edit ServiceNow Notifications

To edit a ServiceNow notification for Chef Automate:

1. From the Notifications page, select the notification name to open its detail page.
1. Edit the notification type, name, failure type, or URL.
1. Select the **Save Notification** button to save the ServiceNow notification.

### Delete ServiceNow Notifications

To delete a ServiceNow notification for Chef Automate:

1. From the Notifications page, select **Delete** from the menu at the end of the table row.
1. Confirm that you wish to permanently delete this notification.

## Slack Notifications

### Add a Slack Notification

To add a Slack notification for Chef Automate:

![Slack Navigation](/images/automate/notifications-navigation.png)

1. In the **Settings** tab, navigate to the _Notifications_ page in the sidebar.
1. Select **Create Notification**.
1. Select **Slack**.
1. Enter a unique notification name.
1. Select the failure type to be notified on from the drop-down menu. Current options are Chef Infra Client run or InSpec scan

> 1. Get your Slack webhook address, which opens an external Slack site.

Reviewer 2: ```suggestion
1. Get your Slack webhook address from [here](https://api.slack.com/messaging/webhooks).
```
